_pages/en/it-picture-long-term.md

```diff
@@ -10,105 +10,61 @@ permalink: /it-picture-long-term.html
 
-Canadians who do not opt out benefit seamlessly from GC services.
-For example from their:
+As such, the relationship between "business" and "IT" has moved from a contacting one, to a partnership one (TODO: link to IT Revolution book - war & peace & IT, or a seat at the table). IT is no longer seen as a separate "contracting" organization within a same organization. IT is needed by ESDC to fulfil its mandate. This means IT shares the accountability with ESDC service lines on their ability to:
 
-- Bank(s)
-- Preferred job hunting website
-- Preferred social media platform(s)
-- IoT devices
+- meet their performance requirements (TODO: link to ESDC services performance target),
+- respond to ongoing changes from citizen expectations (TODO: link to ESDC departmental plan or 2018/19 Auditor Report stating that)
+- assure integrity by design (TODO: link to DSP)  by remaining compliant to security, privacy, and information policies.
 
-90% of citizens (33 300 000 people) (TO-DO: Validate this number) have a trusted Canadian profile used to verify/validate identity via the [Pan-Canadian Trust Framework](https://diacc.ca/pan-canadian-trust-framework/).
+At the same time, ESDC service lines, Policy Compliance business units, AND the CFO fully understand the cost and risks of doing IT. This means:
 
-Citizens can receive their benefits seamlessly throughout their lives as major life events trigger the delivery of benefits.
-See the [Human Development Lifecycle (HDLC)](/human-development-life-cycle.html).
+- IT expertise, especially coding, is normalized in ESDC like Finance and Human Resources management are
+- Investments in IT is considered at the design stage of service
+- continuous improvements is built-in operations to manage technical debt in an ongoing manner (TODO: relentless learning and HR implications link)
+- governance enables smaller, more frequent changes as a means to manage risks and investments more accurately (TODO: link to Mark Schwartz video summarizing War and Peace and IT / OCG key initiatives on their policy rest: https://www.canada.ca/en/treasury-board-secretariat/corporate/organization/ocg-key-initiatives.html).
 
-#### Self-Service
+IITB, as the branch providing the corporate functions of IT for ESDC, is a key enabler in allowing ESDC to fulfil its mandate by providing common IT infrastructure and software to the rest of the organization, as well as to the rest of the Federal Government as a whole, allowing ESDC service lines to respond quickly to citizens growing expectations of government services.
 
-The GC (OneGC) Portal provides access to services and benefits (not just ESDC) in a tell us once fashion, as presented in the [GC Digital Operations Strategic Plan: 2018-2022](https://www.canada.ca/en/government/system/digital-government/digital-operations-strategic-plan-2018-2022.html).
+TODO: IITB currently delivers:
```

Comment:
Les lignes au dessous, ceux sont les choses qu'on fait ou les choses on devrait faire?

Car en ce moment on ne soutient pas `For modern software development tooling is a superset of many different subsets of tools`, par exemple

Reply:
Good point. I moved the details under Tooling in section **Empowering Teams to Choose Tools**